cityofaustin/atd-moped
# Refactors timeline component

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10155

- separates timeline into two separate components: phases and milestone
- queries now live in `ProjectTimeline` and `DashboardTimelineModal` and data is passed down into child components (phases and milestones)

## Testing
**URL to test:** 
https://deploy-preview-776--atd-moped-main.netlify.app/

**Steps to test:**
1. go to dashboard, click the phase badge, add or edit phase, badge on dashboard should change accordingly
2. repeat step 1 for milestones meter
3. go to a project, click timeline tab, add or edit phase, badge on summary tab should change accordingly
4. repeat step 2 with milestones

---
#### Ship list
